packages/combo-box/test/lazy-loading.test.js

```diff
@@ -422,6 +422,10 @@ describe('lazy loading', () => {
           comboBox.close();
 
           comboBox.opened = true;
+          // Wait for the async data provider to respond
+          await aTimeout(0);
+          // Wait for items to render
+          await nextFrame();
```

Comment:
`_scrollIntoView` wasn't always working, seems that we need to wait for the data provider here as well. Not sure if we technically need to wait for items to render, but it needs two wait two times to become stable.